test.c

```diff
@@ -12,385 +12,523 @@ int verbose = 0;
 int should_run_havoc = 0;
 
 int inSlackRange(int expected, int actual) {
-  if(actual >= expected) return 1;
-  if(!SLACK) return 0;
-  int minimumAllowed = expected * 0.9;
-  return(actual >= minimumAllowed);
+    if(actual >= expected) return 1;
```

Comment:
Do we really need to change all this formatting? It's hard to read the diff with all these formatting changes. Can we please stick to two spaces for indentation? It will make this PR a lot easier to read (and will keep the git changelog clean)

Reply:
I am using 4 spaces for indentation is just because it is the default setting, I can change it in the next version of PR